### PR TITLE
feat: show all compatible versions

### PR DESCRIPTION
**Pull request by Ushie**

### CODE REVIEW

**Comment by Ushie** [comment]
@PalmDevs @oSumAtrIX Please confirm that the both of you are satisfied with this implementation of #147

**Comment by PalmDevs** [comment]
Do you have any preview of this?

**Comment by Ushie** [comment]
Yes, check the deployment that the ReVanced bot sent

File: src/routes/patches/PatchItem.svelte [hunk 1]
```diff
@@ -36,20 +37,48 @@
...
+			{#if patch.compatiblePackages[0].versions.length > 1}
+				<li class="patch-info button" on:click={() => (showAllVersions = !showAllVersions)}>
+					<img
+						class="expand-arrow"
+						id="expand-versions"
+						style:transform={showAllVersions ? 'rotate(90deg)' : 'rotate(-90deg)'}
+						src="/icons/expand_more.svg"
+						alt="dropdown"
+					/>
+				</li>
```
Comment (oSumAtrIX) [inline src/routes/patches/PatchItem.svelte:79]: I suggest to move  out of <li>. The reason is that the semantics of the <li> here is that it is a chip, but the expand button is not a chip:

Before:
image

After:
image

**Pull request merged**